servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/ViewCatalogTests.java

```diff
@@ -71,11 +69,7 @@ public abstract class ViewCatalogTests<C extends ViewCatalog & SupportsNamespace
 
   protected abstract Catalog tableCatalog();
 
-  @TempDir private Path tempDir;
```

Comment:
Let's leave this as is (for the same reason as mentioned in `CatalogTests.java`).